src/rules/converters/use-default-type-parameter.ts

```diff
@@ -0,0 +1,11 @@
+import { RuleConverter } from "../converter";
+
+export const convertUseDefaultTypeParameter: RuleConverter = () => {
+    return {
+        rules: [
+            {
+                ruleName: "@typescript-eslint/use-default-type-parameter",
```

Comment:
It's actually this rule now: https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/no-unnecessary-type-arguments.md